clang/tools/driver/cc1_main.cpp

```diff
@@ -1189,7 +1189,7 @@ void RemoteCachingOutputs::tryReleaseLLBuildExecutionLane() {
     // this task.
     const char *LLControlFD = getenv("LLBUILD_CONTROL_FD");
```

Comment:
Just saw this. This should be `llvm::sys::Process::GetEnv`

Reply:
Done 👍